.github/workflows/ci.yaml

```diff
@@ -63,6 +63,12 @@ jobs:
           - php: 7.4
             mongo-ext: 1.9.0
             mongo-img: percona/percona-server-mongodb:4.4
+          - php: 8.0
+            mongo-ext: 1.6.0
```

Comment:
https://docs.mongodb.com/drivers/php/#language-compatibility

According to this only ext 1.9 seems to be compatible with php8